movieApp.js

```diff
@@ -0,0 +1,170 @@
+document.addEventListener("DOMContentLoaded",  async ()=>{
```

Comment:
CSS and general design are good but functionality requires some re-work as most of functionalities required in the rubric are not there. Here are some feedbacks: 
1. onchange attribute must be applied to select tag not option tag. It is causing problem in your app. 
2. addEventListener is much preferred way over changing onchange attribute. 
3. Pay attention to data type and what kind of data goes into a function as arguments. I see some instances where you used HTML tag as argument for function which is probably not what you wanted to do. 
4. Keep practicing DOM manipulation and utilizing fetch data since those are very important skills to master as a developer. 

Please reach out to instructional team for further questions or reviews. We will go over the assessment in the class but it is also important for you to work on your own project and also revise this assessment to make all functionalities work.